components/human-body-diagram.js

```diff
@@ -70,97 +35,241 @@ function HumanBodyDiagram({
               )
           : null
       }
+      onMouseOver={
+        highlightOrgans
+          ? (e) => {
+              const organ = e.target.classList[0].replaceAll("-", " ");
+              if (enabledBodyMapFilters.includes(organ)) {
+                highlightOrgans(organ);
+              }
+            }
+          : null
+      }
+      onMouseOut={setHighlightedOrgans ? () => setHighlightedOrgans([]) : null}
```

Comment:
I’m not sure, but can’t `highlightOrgans` be made to handle this operation? Like call `highlightOrgans()`, and then if `highlightOrgans` receives nothing in the `organ` parameter, then it calls `setHighlightedOrgans` with `[]`. That lets you reduce the number of properties for this component by one, and helps put all modification of `highlightedOrgans` in one place. The latter point I think is nice to have when a React state has more than a string or number or boolean.

Reply:
Sounds great!